source/includes/mongodb-compatibility-table-python.rst

```diff
@@ -23,6 +24,7 @@
      - |checkmark|
      - |checkmark|
      - |checkmark|
+     - |checkmark|
```

Comment:
Beta releases are not 'released' via JIRA for the PYTHON project. I think this is because we don't create built distributables for beta releases for upload to python's package index (this is because Python packaging is not particularly smart and beta versions can be installed in lieu of stable releases on systems running older versions of the Python pkg manager `pip`).
AFAIK we only track what issues are fixed in the beta releases using Changelog entries.

Reply:
Note that the latest available version now is `3.11.0rc0`. Please update L19 to indicate the same.

Reply:
How does that look now @prashantmital ? I labeled it "3.11 (release candidate)". We generally only include major and minor versions for the compatibility tables with the assumed understanding that it applies to all the point releases.

Reply:
In that case, can we just remove `(release candidate)` from showing up in this table? The current latest 3.11 release is `3.11.0rc0`. Once 4.4 is GA, we will release 3.11.0. If compatibility is implied solely by major and minor version, there seems to be no reason to not do just put `3.11` here (and avoid an additional PR to remove the `(release candidate)` part later).